sample/main.cpp

```diff
@@ -64,7 +64,7 @@ int main(int argc, char* argv[])
     try
     {
         boost::asio::io_service io_service;
-        Server server(io_service, secret);
+        RadProto::Server server(io_service, secret);
```

Comment:
After the changes in the library, this sample no longer works. You need to update the sample code to make it work like before.